Gruntfile.js

```diff
@@ -13,14 +12,20 @@ module.exports = function (grunt) {
         return fs.statSync(modules_dir + file).isDirectory();
     });
 
-     _.each(modules, function (name) {
+    _.each(modules, function (name) {
+        var module_path = name + '.templates';
```

Comment:
Can you add `Gruntfile.js` to the lint files? Then snake_case vars like this will be picked up